packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -281,7 +291,7 @@ export class ZkEvmProvider implements Provider {
               guardianClient: this.#guardianClient,
               rpcProvider: this.#rpcProvider,
               relayerClient: this.#relayerClient,
-              zkevmAddress: this.#zkEvmAddress!,
+              zkevmAddress: zkEvmAddress,
```

Comment:
Can we update the `zkevmAddress` property that `sendTransaction` accepts to be consistent with the rest of the codebase? E.g. `zkEvmAddress` instead of `zkevmAddress`